system/nagios-status-check/detectors-nagios-status-check.tf

```diff
@@ -0,0 +1,27 @@
+resource "signalfx_detector" "status_check" {
+  name = "${join("", formatlist("[%s]", var.prefixes))}[${var.environment}] Nagios check status"
+
+  program_text = <<-EOF
+        signal = data('nagios_state.state', filter=${module.filter-tags.filter_custom})${var.status_check_aggregation_function}.${var.status_check_transformation_function}(over='${var.status_check_transformation_window}').publish('signal')
+        detect(when(signal >= threshold(1)) and when(signal < 2)).publish('WARN')
+        detect(when(signal >= threshold(2))).publish('CRIT')
```

Comment:
is there no "Unknown" status ?

Reply:
Yes there is, I was wondering if it worth it but I just added it with the severity "Critical" as there is no unknown in SignalFx.